test/units/testsuite-74.pstore.sh

```diff
@@ -125,6 +125,22 @@ start_pstore() {
     journalctl --sync
 }
 
+check_journal() {
+    local file="${1:?}"
+
+    for _ in {0..9}; do
```

Comment:
Is this loop necessary? IIUC the commit message, the root cause is that there should exist expected log entries, but these may not have unit field.
Why not simply drop `-u systemd-pdtore.service`?

Reply:
I added it mainly as a yet another safety net for slower CIs, but I'm not completely sure if such scenario can occur. I can definitely drop it if it's unnecessary.